### DISCUSSION TITLE
V1又又又废掉了几个功能

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#3412] [综合讨论 / General]
![图片](https://user-images.githubusercontent.com/81268244/172386013-46d3fc17-5f2f-48ef-8acc-70da17760b83.png)
![图片](https://user-images.githubusercontent.com/81268244/172386015-865fcd62-3e4e-4083-96b7-6376a6752326.png)
如图，简化评论区功能废了一半，评论链接复制功能彻底废了

刚准备升级V2，但看了看issues，好像V2不挂tz不能用了？

**Comment by the1812** [comment]
是的

**Comment by the1812** [comment]
不过评论区貌似是今天改版的, v2 目前一样挂

**Reply by iseek0** [reply]
我说呢。。评论区发送时间不在右上角，复制评论链接没法显示

**Reply by Tinhone** [reply]
奇了怪了，有些脚本没挂欸

**Comment by RingingVolcano** [comment]
因为jsdelivr被墙的原因其实现在v1不挂代理应该也不太行了（

**Reply by Tinhone** [reply]
其实V1不挂代理也可以用，歪瑞古德 ( ﾟ∀。)

**Reply by hmgdd101** [reply]
V1的离线版是可以的
